### PR TITLE
Add support for APIM 4.1.0-beta in Docker images

### PR DESCRIPTION
**Pull request by IsuruMaduranga**
## Purpose
To support APIM 4.1.0-beta in dockerfiles.

## Related PRs
https://github.com/wso2/docker-apim/pull/453
https://github.com/wso2/docker-apim/pull/454

### CODE REVIEW

File: dockerfiles/ubuntu/apim/Dockerfile [hunk 1]
```diff
@@ -76,7 +76,7 @@ RUN echo Verifying install ... \
     && echo Complete.
 
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-apim/releases/tag/v4.0.0.4"
+      com.wso2.docker.source="https://github.com/wso2/docker-apim/releases/tag/v4.1.0"
```
Comment (RakhithaRR) [inline dockerfiles/ubuntu/apim/Dockerfile:79]: Should be 4.1.0.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,29 +1,13 @@
...
-
-- Upgrade MySQL version used in Docker Compose resources - `5.7.34`.
-
-### Removed
-
-- WSO2 API Manager Analytics Dashboard and Worker related Docker Compose resources.
+- Update all Docker resources to support WSO2 API Manager version `4.1.0`.
+- Update all Docker Compose resources to supprt WSO2 API Manager version `4.1.0`.
+- Update Docker Compose resources for the deployment of WSO2 APIM with MI to support Micro Integrator version `4.1.0.0`.
+- Update IS extentions to latest version `1.2.10` and mount wso2carbon and client-truststore keystores with the latest wso2carbon certificate in Identity Server as Key Manager with Choreo Analytics deployment setup.
```
Comment (RakhithaRR) [inline CHANGELOG.md:13]: This should be 1.4.2 right?

Comment (IsuruMaduranga) [reply CHANGELOG.md]: Updated

File: dockerfiles/ubuntu/apim/Dockerfile (hunk 1 above)
Comment (IsuruMaduranga) [reply dockerfiles/ubuntu/apim/Dockerfile]: Updated

**Review by RakhithaRR — APPROVED** [review]

**Pull request merged**